scripts/googleAppScripts/SheetToGithubIssue.gs

```diff
@@ -1,100 +1,166 @@
+    var issueBody = `### Initiative Name
+${initiativeName}
+
+### Initiative Description
+${initiativeDescription}
+
+### Initiative Details
+${initiativeDetails}
+
+### Initiative Validation Details
+${initiativeValidationDetails}
+
+### Initiative Category
+${initiativeCategory}
+
+### Initiative Main URL
+${initiativeMainURL}
+
+### Initiative Logo
+${initiativeLogo}
+
+### Initiative Website
+${initiativeWebsite}
+
+### Initiative Phone Number
+${initiativePhoneNumber}
+
+### Initiative E-Mail
+${initiativeEmail}
+
+### Initiative WhatsApp
+${initiativeWhatsApp}
+
+### Initiative Telegram
+${initiativeTelegram}
+
+### Initiative Drive
+${initiativeDrive}
+
+### Initiative Form
+${initiativeForm}
+
+### Initiative Document
+${initiativeDocument}
+
+### Initiative Discord
+${initiativeDiscord}
+
+### Initiative Facebook
+${initiativeFacebook}
+
+### Initiative Instagram
+${initiativeInstagram}
+
+### Initiative TikTok
+${initiativeTikTok}
+
+### Initiative X (Twitter)
+${initiativeTwitter}
+
+### Initiative LinkedIn
+${initiativeLinkedIn}
+
+### Initiative YouTube Page / Channel
+${initiativeYouTube}
+
+### Initiative Donation Link
+${initiativeDonationLink}
+
+### Initiative Tags
+${initiativeTags}`;
+
+    var issueTitle = `[NEW-INITIATIVE]: ${initiativeName}`;
 
     // Call the function to create a GitHub issue
     createGithubIssue(issueTitle, issueBody);
 }
 
 function createGithubIssue(title, body) {
-  var scriptProperties = PropertiesService.getScriptProperties();
-  var githubToken = scriptProperties.getProperty('GITHUB_TOKEN');
-  var repo = '4tals/LinksForIsrael';
-  var url = 'https://api.github.com/repos/' + repo + '/issues';
-
-  var payload = {
-    title: title,
-    body: body,
-    labels: ['new-initiative-request']
-  };
-
-  var options = {
-    method: 'post',
-    headers: {
-      Authorization: 'token ' + githubToken,
-      Accept: 'application/vnd.github.v3+json'
-    },
-    contentType: 'application/json',
-    payload: JSON.stringify(payload),
-    muteHttpExceptions: true // This will prevent the script from throwing an exception on HTTP errors
-  };
-
-  try {
-    var response = UrlFetchApp.fetch(url, options);
-    Logger.log(response.getContentText());
-  } catch (error) {
-    Logger.log('Error creating GitHub issue: ' + error.toString());
-  }
+    var personalGithubToken = PropertiesService.getUserProperties().getProperty('GITHUB_TOKEN');
+    var repo = '4tals/LinksForIsrael';
+    var url = 'https://api.github.com/repos/' + repo + '/issues';
+
+    var payload = {
+        title: title, body: body, labels: ['new-initiative-request']
+    };
+
+    var options = {
+        method: 'post', headers: {
+            Authorization: 'Bearer ' + personalGithubToken, Accept: 'application/vnd.github+json'
+        }, contentType: 'application/json', payload: JSON.stringify(payload),
+    };
+
+    try {
+        var response = UrlFetchApp.fetch(url, options);
+        Logger.log(response.getContentText());
```

Comment:
Let's be consistent and use `console` everywhere

Reply:
Done.

Reply:
minor: I still see a copule of `Logger` uses (in the test methof)